testproject-tools-integration/Assets/Tests/Runtime/testproject.toolsintegration.runtimetests.asmdef

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "TestProject.ToolsIntegration.RuntimeTests",
+    "rootNamespace": "TestProject.ToolsIntegration.RuntimeTests",
+    "references": [
+        "Unity.Netcode.Runtime",
+        "Unity.Netcode.RuntimeTests",
+        "Unity.Multiplayer.MetricTypes",
+        "Unity.Multiplayer.NetStats",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
```

Comment:
nit: your runtime tests probably shouldn't depend on things from UnityEditor, right? Not exactly sure the context of "runtime" tests for tools tests. If this dll is expected to work in a standalone build, it won't with this dependency.

Reply:
We can deal with it later. Can you make sure there's a tracking issue filed somewhere so we don't forget about it? This ended up biting us pretty badly when we enabled platform CI for mobile & consoles.

Reply:
https://jira.unity3d.com/browse/MTT-1383 done!